lib/gracenote/request/option.rb

```diff
@@ -0,0 +1,44 @@
+module Gracenote
+  module Request
+    class Option
+      include Helper
+
+      def initialize(xml, options = {})
+        @xml = xml
+
+        @options = {
+          "SELECT_EXTENDED" => %w(
+            COVER
+            REVIEW
+            ARTIST_OET
+            MOOD
+            TEMPO
+            ARTIST_BIOGRAPHY
+            ARTIST_IMAGE
+            CONTENT
+            LINK
+          ),
+          "SELECT_DETAIL" => %w(
+            GENRE:3LEVEL
+            MOOD:2LEVEL
+            TEMPO:3LEVEL
+            ARTIST_ORIGIN:4LEVEL
+            ARTIST_ERA:2LEVEL
+            ARTIST_TYPE:2LEVEL
+          )
+        }.merge(options)
+      end
+
+      def render
+        @options.each { |k, v| render_option(k, v) }
+      end
+
+      def render_option(key, value)
+        @xml.OPTION {
```

Comment:
__[rubocop]__
* Avoid using `{...}` for multi-line blocks. [:ref](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)